src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.CodeGeneration.targets

```diff
@@ -14,27 +26,32 @@
     AssemblyFile="$(RazorSdkBuildTasksAssembly)"
     Condition="'$(RazorSdkBuildTasksAssembly)' != ''" />
 
-  <!-- 
+  <!--
     Consider these properties to be private to this targets file. The main Razor SDK should define all of the properties
-    that we use to pass data back and forth. 
+    that we use to pass data back and forth.
   -->
   <PropertyGroup>
+    <!-- Continue setting this property to maintain compat with legacy Razor.Design -->
+    <IsRazorCompilerReferenced>true</IsRazorCompilerReferenced>
```

Comment:
So if you are using the 3.0 SDK on a 2.0 project ..... how does this behave?

Reply:
This file only gets imported if a property is set (set by default for netcoreapp 3.0 or later projects).